modules/microshift-cni-customization-matrix.adoc

```diff
@@ -0,0 +1,47 @@
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-nw-customization-matrix_{context}"]
+= {microshift-short} networking customization matrix
+
+The following table summarizes the status of networking features and capabilities that are either present as defaults, supported for configuration, or not available with the {microshift-short} service:
+
+.{microshift-short} networking capabilities and customization status
+[cols="50%,20%,30%",options="header"]
+|===
+|Network feature|Availability|Customization supported
+
+|Advertise address|Yes|Yes ^[1]^
+
+|Kubernetes network policy|Yes|Yes
+
+|Kubernetes network policy logs|Not available|N/A
+
+|Load balancing|Yes|Yes
+
+|Multicast DNS|Yes|Yes ^[2]^
+
+|Network proxies|Yes ^[3]^|CRI-O
+
+|Network performance|Yes|MTU configuration
+
+|Egress IPs|Not available|N/A
+
+|Egress firewall|Not available|N/A
+
+|Egress router|Not available|N/A
+
+|Firewall|No ^[4]^|Yes
+
+|Hardware offloading|Not available|N/A
+
+|Hybrid networking|Not available|N/A
+
+|IPsec encryption for intra-cluster communication|Not available|N/A
+
+|IPv6|Not available|N/A
```

Comment:
Since we are explicitly defining whats supported or not. Does Mentioning IPv6 not available implies that Dual stack is not supported? 

rest technicalities looks good as most of the part has been reviewed in last releases

Reply:
Or it probably implies that IPv6 is not available in terms of dual stack and single stack IPv6

Reply:
Clarified in a footnote that IPv6 is not supported in any config.